packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/DeferredEventEmitter.java

```diff
@@ -40,6 +42,8 @@
     private static final String EVENT_NAME_MEDIA_SAVE = "mediaSave";
     private static final String EVENT_NAME_MEDIA_REPLACE_BLOCK = "replaceBlock";
 
+    private static final String EVENT_FEATURED_IMAGE_ID_CURRENT = "featuredImageIdCurrent";
```

Comment:
(Another nit?) Now that on the JS side we use the core-data Store for featured image id storage, I can't quite be sure what we mean when we say "Current". I know that [it was me not long before](https://github.com/WordPress/gutenberg/pull/28854#discussion_r598733588) that proposed the name, but back then I felt that the source of truth is the data we keep in the native side. That's kinda still true but when we'll add the functionality of setting the featured image from the Image block, the source of truth will the be image block itself or the updated core-data store if we go that way. In that case, what's "current"?

Long story short, wdyt about renaming to `featuredImageIdNativeUpdated`?

Reply:
Your reasoning behind the name change makes sense to me here, I've gone ahead to make that update in this PR and the Android update will follow soon.